Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -613,12 +613,13 @@ extension NavigatorIndex {
         
         /// Index a single render `RenderNode`.
         /// - Parameter renderNode: The render node to be indexed.
-        public func index(renderNode: RenderNode) throws {
+        /// - Parameter ignoringLanguage: Whether language variants should be ignored when indexing this render node.
+        public func index(renderNode: RenderNode, ignoringLanguage: Bool = false) throws {
```

Comment:
Introducing this parameter remains source stable because it has a default value, so exiting calls to `.index(renderNode: renderNode)` .

However, if we later wanted to change the name of this parameter, _that_ would be source breaking. 

Considering that, are we happy with `ignoringLanguage` or do we want to include the word "variants" somewhere in this parameter name?

Reply:
Since both the parameter documentation and the code comment above where this value is checked mentions variants, I'm pretty sure that future us will understand what it's for regardless.

Reply:
What do you think? I don't have a strong opinion.

Reply:
I'm also fine either way.